### PR TITLE
fix: replaced deprecated MKMapView showAnnotations with edgePadding f…

### PR DESCRIPTION
**Pull request by theonetheycallneo**
…unction

### Does any other open PR do the same thing?

(please answer here)

### What issue is this PR fixing?

(please link the issue here)

### How did you test this PR?

(please answer here)

### CODE REVIEW

**Comment by rborn** [comment]
LGTM, 

Is it breaking anything for the older versions?
Also how does this affect android? Do we need to do this on the android side too?

@alvelig 🐽

File: lib/ios/AirMaps/AIRMapManager.m [hunk 1]
```diff
@@ -276,8 +276,9 @@ - (UIView *)view
             CGFloat bottom = [RCTConvert CGFloat:edgePadding[@"bottom"]];
             CGFloat left = [RCTConvert CGFloat:edgePadding[@"left"]];
 
-            [mapView showAnnotations:filteredMarkers edgePadding:UIEdgeInsetsMake(top, left, bottom, right) animated:animated];
-            
+            [mapView setVisibleMapRect:mapView.visibleMapRect edgePadding:UIEdgeInsetsMake(top, left, bottom, right) animated:animated];
```
Comment (alvelig) [inline lib/ios/AirMaps/AIRMapManager.m]: Why do we need this line?

File: lib/ios/AirGoogleMaps/AIRGoogleMapManager.m [hunk 2]
```diff
@@ -236,7 +236,8 @@ - (UIView *)view
       CGFloat bottom = [RCTConvert CGFloat:edgePadding[@"bottom"]];
       CGFloat left = [RCTConvert CGFloat:edgePadding[@"left"]];
 
-      [mapView animateWithCameraUpdate:[GMSCameraUpdate fitBounds:bounds withPadding:55.0f withEdgeInsets:UIEdgeInsetsMake(top, left, bottom, right)]];
+      // [mapView animateWithCameraUpdate:[GMSCameraUpdate fitBounds:bounds withPadding:55.0f withEdgeInsets:UIEdgeInsetsMake(top, left, bottom, right)]];
```
Comment (alvelig) [inline lib/ios/AirGoogleMaps/AIRGoogleMapManager.m]: Maybe remove the comment then?

**Comment by marcelkalveram** [comment]
@alvelig maybe you wann have a look at my PR where I fixed two function calls that were causing the build to fail: https://github.com/react-community/react-native-maps/pull/2524

**Comment by rborn** [comment]
@theonetheycallneo I merged #2524 in favor of this one - hotfix was required as the app wouldn't compile. Thanks for the PR ❤️

**Pull request closed without merging**